src/jdk.compiler/share/classes/com/sun/tools/javac/resources/compiler.properties

```diff
@@ -155,6 +155,10 @@ compiler.err.annotation.value.not.allowable.type=\
 compiler.err.expression.not.allowable.as.annotation.value=\
     expression not allowed as annotation value
 
+# 0: type, 1: name
+compiler.err.annotation.unrecognized.attribute.name=\
+    annotation @{0} has an unrecognizable attribute named ''{1}''
```

Comment:
"unrecognizable" seems a strange word here;  would "unknown" be a better word?

Reply:
sure will fix